V3-Upgrade-Guide.md

```diff
@@ -46,6 +46,9 @@ Therefore, all SAP Cloud SDK for JavaScript libraries will switch to node 18 as
 If you run an older (<18) node version, please update to a newer version.
 You can find a list of breaking changes in the news section of the node.js [website](https://nodejs.org/en/blog/).
 
+Please not, in the `tsconfig.json`, we updated the configuration, so it looks like `"module": "Node16"`.
+As `Node18` is not a valid value of `module` for the time being, we might update it later.
```

Comment:
I don't know if they'll do that.. is this sentence important?

Reply:
This is a note for ourselves and the users, as you might be confused by the "16".
@marikaner , what's your opinion?

Reply:
[pp] Also - I agree it is not releavant for the upgrade guide. Maybe rather a compatibility note.